File: src/platform/backend/routes/user_routes.py
from django.urls import path
from src.backend.views import user_views  # Ajusta la ruta según tu proyecto

urlpatterns = [
    path('users/', user_views.user_list, name='user-list'),
    path('users/<int:pk>/', user_views.user_detail, name='user-detail'),
    path('users/create/', user_views.user_create, name='user-create'),
    path('users/update/<int:pk>/', user_views.user_update, name='user-update'),
    path('users/delete/<int:pk>/', user_views.user_delete, name='user-delete'),
]

File: src/platform/backend/routes/user_routes.js
const express = require('express');
const router = express.Router();
const userController = require('../controllers/user_controller'); // Ajusta la ruta según tu proyecto

router.get('/users', userController.userList);
router.get('/users/:id', userController.userDetail);
router.post('/users', userController.userCreate);
router.put('/users/:id', userController.userUpdate);
router.delete('/users/:id', userController.userDelete);

module.exports = router;
